app/Enums/UserContext.php

```diff
@@ -15,18 +15,18 @@ public static function labels(): array
         return [
             'individual' => __('Individual'),
             'organization' => __('Community Organization'),
-            'regulated-organization' => __('Regulated Organizations: Business, Federal Government and Public Sector Organizations'),
+            'regulated-organization' => __('Federally Regulated Organizations'),
```

Comment:
I don't think this should be plural if the others aren't.

Reply:
The copy suggestions say it in plural, should we just make it singular?

Reply:
Yeah I think the copy suggestion is wrong. Thoughts, @cherylhjli / @michelled?

Reply:
The question is:

**Tell us who you are joining the Accessibility Exchange as**

The answers are:

- Individual
- Community Organization
- Federally Regulated Organization

If the last one is plural it doesn't fit